tests/Feature/GithubAppTest.php

```
@@ -77,6 +77,11 @@ public function test_manifest_callback_creates_app(): void
             'app_id' => 12345,
             'app_slug' => 'vito-test',
         ]);
+
+        Http::assertSent(function ($request) {
+            return $request->url() === 'https://api.github.com/app-manifests/test-code/conversions'
+                && $request->body() === '';
+        });
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Http' in method 'test_manifest_callback_creates_app'. (undefined)

(StaticAccess)